pinc/access_change_emails.inc

```diff
@@ -56,6 +56,96 @@ function pp_access_change_email($user, $access_change)
     configure_gettext_for_user();
 }
 
+function p2_access_change_email($user, $access_change)
+{
+    global $site_name, $site_abbreviation, $Round_for_round_id_;
+
+    $daily_limit = $Round_for_round_id_['P2']->daily_page_limit;
+
+    // only send emails on grant
+    if ($access_change != "grant") {
+        return;
+    }
+
+    // configure gettext to translate user email
+    configure_gettext_for_user($user);
+
+    // TRANSLATORS: %s is the site abbreviation (eg: 'DP')
+    $subject = sprintf(_("%s: Welcome to P2 Proofreading"), $site_abbreviation);
+
+    $message[] = sprintf(_("Hello %s,"), $user->username);
+
+    $message[] = _("Congratulations! You may now work in P2, the second proofreading round.");
+
+    $message[] = "==" . _("Compared to P1, what remains the same?") . "==";
+
+    $message[] = _("The Proofreading Guidelines are exactly the same, and you still need to comply with the Project Comments and Project Discussion guidance for every project.");
+
+    $message[] = _("Please keep aiming to leave every page as perfect as you can.");
+
+    $message[] = "==" . _("What is expected in P2?") . "==";
```

Comment:
Since I was thinking the same thing as I was crafting the P2 email in the first place, I went ahead and changed the instances there were. Definitely easier to parse.